AboutIOFDev.rdf

```diff
@@ -1,37 +1,47 @@
-	xmlns:xsd="http://www.w3.org/2001/XMLSchema#"
-	xmlns:iof-av="https://spec.industrialontologies.org/ontology/core/meta/AnnotationVocabulary/"
-	xmlns:dcterms="http://purl.org/dc/terms/">
-	
-  	<owl:Ontology rdf:about="https://spec.industrialontologies.org/ontology/AboutIOFDev/">
-		<rdfs:label xml:lang="en-US">About IOF Development</rdfs:label>
-		<dcterms:abstract>This ontology defines the scope of development maturity level IOF ontologies.</dcterms:abstract>
-		<dcterms:license rdf:datatype="&xsd;anyURI">http://opensource.org/licenses/MIT</dcterms:license>
-		<dcterms:title>Industrial Ontology Foundry (IOF) Ontology - Development</dcterms:title>
-		<iof-av:copyright>Copyright (c) 2022, 2023, Open Applications Group</iof-av:copyright>
-		<owl:versionIRI rdf:resource="https://spec.industrialontologies.org/ontology/202301/AboutIOFDev/"/>
-		<iof-av:maturity rdf:resource="&iof-av;Provisional"/>
-		<owl:imports rdf:resource="https://spec.industrialontologies.org/ontology/core/Core/"/>
-		<owl:imports rdf:resource="https://spec.industrialontologies.org/ontology/core/meta/AnnotationVocabulary/"/>
-		<owl:imports rdf:resource="https://spec.industrialontologies.org/ontology/core/commonstocoremapping/meta/MappingAnnotationVocabularyToCommons/"/>
-		<owl:imports rdf:resource="https://spec.industrialontologies.org/ontology/core/commonstocoremapping/MappingCommonsToIOF/"/>
-		<owl:imports rdf:resource="https://spec.industrialontologies.org/ontology/maintenance/MaintenanceReferenceOntology/"/>
-		<owl:imports rdf:resource="https://spec.industrialontologies.org/ontology/supplychain/SupplyChain/"/>
-	</owl:Ontology>
+  xmlns:dct="http://purl.org/dc/terms/"
+  xmlns:owl="http://www.w3.org/2002/07/owl#"
+  xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+  xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
+  xmlns:skos="http://www.w3.org/2004/02/skos/core#"
+  xmlns:xsd="http://www.w3.org/2001/XMLSchema#"
+  xmlns:iof-av="https://spec.industrialontologies.org/ontology/core/meta/AnnotationVocabulary/"
+  xmlns:dcterms="http://purl.org/dc/terms/">
+  
+    <owl:Ontology rdf:about="https://spec.industrialontologies.org/ontology/AboutIOFDev/">
+    <rdfs:label xml:lang="en-US">About IOF Development</rdfs:label>
+    <dcterms:abstract>This ontology defines the scope of development maturity level IOF ontologies.</dcterms:abstract>
+    <dcterms:license rdf:datatype="&xsd;anyURI">http://opensource.org/licenses/MIT</dcterms:license>
+    <dcterms:title>Industrial Ontology Foundry (IOF) Ontology - Development</dcterms:title>
+    <iof-av:copyright>Copyright (c) 2022, 2023, Open Applications Group</iof-av:copyright>
+    <owl:versionIRI rdf:resource="https://spec.industrialontologies.org/ontology/202301/AboutIOFDev/"/>
+    <iof-av:maturity rdf:resource="&iof-av;Provisional"/>
+
+    <!-- load from the cache -->
+    <owl:imports rdf:resource="https://spec.industrialontologies.org/iof/ontology/cache/bfo/2020/bfo.rdf"/>
```

Comment:
Given the current version of the catalog-v001.xml file this should read now http://purl.obolibrary.org/obo/bfo/2020/bfo.owl.

Similar remarks about other imports - you should use the ontology IRIs you added to the catalog-v001.xml file.

Reply:
The goal is to make loading an "about" file from an IRI correctly load BFO and the exact versions of Commons that are needed. Therefore, line 32 above should not change.

Reply:
> The goal is to make loading an "about" file from an IRI correctly load BFO and the exact versions of Commons that are needed. Therefore, line 32 above should not change.

Well, I am not sure what you mean by "exact version", but as of now, the import statements in quesion will fail - see, for example, the snip below.

The reason for this is the difference betweeen the IRIs of the imported ontologies in the about file(s) and in the catalog-v001.xml file in [111744a](https://github.com/iofoundry/ontology/pull/77/commits/111744a8533af90e89edf04acb454d15d0899bb8) comit. So either you need to change the about file or the catalog-v001.xml.

My recommendation is based on the following idea of cache: you cache an ontology in case the IRI of this ontology will not resolve. So if it does resolve, the ontology is taken from the resolution site; otherwise, it is taken from the IOF cache folder.

![image](https://github.com/iofoundry/ontology/assets/11171688/c61ac36a-8d5c-4179-ac48-a88ddaeea906)

Reply:
I may have missed one of the corrections we were making yesterday. I'll check. 

If the loads for the catalog or the about files are not coming from cache, please fix them. 

The about files are now referencing the base IRI that should be available once it becomes the default version. We need, so the about files can only be tested once we merge and the cache path is no only in a branch. 

We will test once we have finished the merge and force push if necessary if the about files don't work.

Reply:
The missing imports are because we are prepping for a release. The files _should_ be there once we have completed the merge and tagged.

Reply:
> The missing imports are because we are prepping for a release. The files _should_ be there once we have completed the merge and tagged.

Let me go back to https://github.com/iofoundry/ontology/pull/77#discussion_r1507244203.

Once you have 
<uri id="289c79d8-24a7-4ad6-a7d0-49da596ebe07" name="http://purl.obolibrary.org/obo/bfo.owl" uri="./cache/bfo/2020/bfo.rdf"/>
in the catalog file and
<owl:imports rdf:resource="http://purl.obolibrary.org/obo/bfo.owl"/>
everything you need will be in place right aways - you will not have to wait for the release.

I can see that for some reason you may want to freeze a particular version of bfo and use https://spec.industrialontologies.org/iof/ontology/cache/bfo/2020/bfo.rdf for this purpose, but this would require a change (back) in the catalog file and the change in cache/bfo/2020/bfo.rdf file itself so that the latter is identified by means of https://spec.industrialontologies.org/iof/ontology/cache/bfo/2020/bfo.rdf.